runtime/oak/src/lib.rs

```diff
@@ -673,11 +673,11 @@ parameter_types! {
 	/// Default percent of inflation set aside for parachain bond every round
 	pub const DefaultParachainBondReservePercent: Percent = Percent::from_percent(30);
 	/// Blocks per round
-	pub const DefaultBlocksPerRound: u32 = 2 * HOURS;
+	pub const DefaultBlocksPerRound: u32 = 6 * HOURS;
 	/// Minimum stake required to become a collator
-	pub const MinCollatorStk: u128 = 400_000 * DOLLAR;
+	pub const MinCollatorStk: u128 = 800_000 * DOLLAR;
 	/// Minimum stake required to be reserved to be a candidate
-	pub const MinCandidateStk: u128 = 2_000_000 * DOLLAR;
+	pub const MinCandidateStk: u128 = 800_000 * DOLLAR;
 }
 impl pallet_parachain_staking::Config for Runtime {
```

Comment:
@nvengal - please review all the parachain-staking changes. See Jira tix for requirements: https://oak-network.atlassian.net/browse/ENG-621

Reply:
left some other comments but 👍 on parachain staking runtime config